### PR TITLE
Update: external evaluation of Validated Streams milestone 2

### PR DESCRIPTION
**Pull request by arahangua**
As all outstanding issues have been resolved, I would like to submit an updated evaluation for this [milestone](https://github.com/w3f/Grant-Milestone-Delivery/pull/881). 

Thanks!

### CODE REVIEW

**Comment by semuelle** [comment]
Hi @arahangua. Thanks again for your work. I can confirm that all the issues are fixed and everything works as expected. I will merge your PR and forward your info for your KSM reward.

Nice work using commits to refer to files and ensuring that the tests are running. With regard to tarpaulin, I find the percentages it reports not quite reliable. There is a fair bit of code in a pallet that does not need to be tested, like weights. Looking at the raw numbers, the coverage for the pallet is closer to 50%. I'll make a change suggestion to reflect that. And, just to completeness: we like to leave initial feedback in the document and add updates chronologically. It's true that the previous evaluation can be read by going through the history of the evaluation file, but the process is a lot easier to track when it's all in one file.

File: evaluations/validated-streams_2_arahangua.md [hunk 1]
```diff
@@ -1,501 +1,34 @@
...
+| **0b.** | Documentation | <ul><li>[x] </li></ul> | [README](https://github.com/comrade-coop/validated-streams/blob/6f9478b84ea5730fceaae50c04b62e1ecee88fed/README.md) | easy to follow description with a schematic. |
+| **0c.** | Testing and Testing Guide | <ul><li>[x] </li></ul> | [Running tests](https://github.com/comrade-coop/validated-streams/blob/6f9478b84ea5730fceaae50c04b62e1ecee88fed/README.md#testing) <br> [proofs](https://github.com/comrade-coop/validated-streams/blob/6f9478b84ea5730fceaae50c04b62e1ecee88fed/consensus/src/proofs/tests.rs) <br> [events](https://github.com/comrade-coop/validated-streams/blob/b5a9437b2e59881a64c63596ded9144f72a39ad5/consensus/src/events/tests.rs) <br> [pallet](https://github.com/comrade-coop/validated-streams/blob/a8a55ab7f2f16401c8aefefccdcdfa4638ddb39a/pallet/src/tests.rs) | <ul><li> with 'on-chain proofs' : `cargo tarpaulin` reports ~27% test coverage for 'consensus' crate with 19 tests and ~7% coverage for 'pallet' crate with 8 tests. </li></ul><ul><li> Without 'on-chain proofs' : `cargo tarpaulin` reports ~27% test coverage for the 'consensus' crate with 19 tests and ~5% coverage for the 'pallet' with 3 tests. |
```
Comment (semuelle) [inline evaluations/validated-streams_2_arahangua.md:13]:
```suggestion
| **0c.** | Testing and Testing Guide | <ul><li>[x] </li></ul> | [Running tests](https://github.com/comrade-coop/validated-streams/blob/6f9478b84ea5730fceaae50c04b62e1ecee88fed/README.md#testing) <br> [proofs](https://github.com/comrade-coop/validated-streams/blob/6f9478b84ea5730fceaae50c04b62e1ecee88fed/consensus/src/proofs/tests.rs) <br> [events](https://github.com/comrade-coop/validated-streams/blob/b5a9437b2e59881a64c63596ded9144f72a39ad5/consensus/src/events/tests.rs) <br> [pallet](https://github.com/comrade-coop/validated-streams/blob/a8a55ab7f2f16401c8aefefccdcdfa4638ddb39a/pallet/src/tests.rs) | <ul><li> with 'on-chain proofs' : `cargo tarpaulin` reports ~60% test coverage for 'consensus' crate with 19 tests and ~70% coverage for 'pallet' crate with 8 tests. </li></ul><ul><li> Without 'on-chain proofs': `cargo tarpaulin` reports ~60% test coverage for the 'consensus' crate with 19 tests and ~50% coverage for the 'pallet' with 3 tests. |
```

File: evaluations/validated-streams_2_arahangua.md [hunk 2]
```diff
@@ -1,501 +1,34 @@
 # Evaluation
 
-- **Status:** In Progress
+- **Status:** Accepted
 - **Application Document:** https://github.com/w3f/Grants-Program/blob/master/applications/validated-streams.md
 - **Milestone:** 2 
 - **Kusama Identity:** [HZrUkonEQc9tReQNndBTML5NbYWBaj8KGaHUGGxwq3jP9PY](https://explorer.polkascan.io/kusama/account/HZrUkonEQc9tReQNndBTML5NbYWBaj8KGaHUGGxwq3jP9PY)
 - **Previously successfully merged evaluation:** All by semuelle
```
Comment (semuelle) [inline evaluations/validated-streams_2_arahangua.md:7]:
```suggestion
- **Previously successfully merged evaluation:** none
```

**Review by semuelle — CHANGES_REQUESTED** [review]

**Comment by RouvenP** [comment]
hi @arahangua we just transferred the KSM

**Pull request merged**